spec/features/user.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :user do
+    sequence(:username) { |n| "person#{n}" }
```

Comment:
Looks like you removed username... which is great but you need to remove it from teh factory, too